Jenkinsfile

```diff
@@ -0,0 +1,74 @@
+#!groovy
+
+pipeline {
+  agent {
+    docker {
+        image 'dealii/dealii:v8.5.1-gcc-mpi-fulldepscandi-debugrelease'
+	label 'has-docker'
+    }
+  }
+
+  parameters {
+    booleanParam(defaultValue: false, description: 'Is the pull request approved for testing?', name: 'TRUST_BUILD')
+  }
+
+  stages {
+    stage('Check indentation') {
+      steps {
+        sh './doc/indent'
+        sh 'git diff | tee changes-astyle.diff'
+        archiveArtifacts artifacts: 'changes-astyle.diff', fingerprint: true
+        sh 'git diff --exit-code --name-only'
+        }
+    }
+
+    stage ("Check execution") {
+      when {
+	allOf {
+            environment name: 'TRUST_BUILD', value: 'false' 
+            not {branch 'master'}
+            not {changeRequest authorEmail: "rene.gassmoeller@mailbox.org"}
+            not {changeRequest authorEmail: "timo.heister@gmail.com"}
+            not {changeRequest authorEmail: "bangerth@colostate.edu"}
+            not {changeRequest authorEmail: "judannberg@gmail.com"}
+            not {changeRequest authorEmail: "ja3170@columbia.edu"}
+            not {changeRequest authorEmail: "john.naliboff@gmail.com"}
+            not {changeRequest authorEmail: "menno.fraters@outlook.com"}
+            not {changeRequest authorEmail: "acglerum@gfz-potsdam.de"}
+	    }
+      }
+      steps {
+	  echo "Please ask an admin to rerun jenkins with TRUST_BUILD=true"
+	    sh "exit 1"
+      }
+    }
+
+    stage('Build') {
+      steps {
+        sh '''
+          mkdir -p build-gcc-fast
```

Comment:
this means the build directory is inside the workspace and so will write outside the container. Maybe not a good idea? In my current setup the docker instance I am building a separate build directory.